pootle/core/initdb.py

```diff
@@ -201,12 +213,11 @@ def create_pootle_permission_sets(self):
     def require_english(self):
         """Create the English Language item."""
         criteria = {
-            'code': "en",
             'fullname': u"English",
             'nplurals': 2,
             'pluralequation': "(n != 1)",
         }
-        en, created = self._create_object(Language, **criteria)
+        en, created = self._create_object(Language, code='en', defaults=criteria)
```

Comment:
It's unrelated, but necessary I think. 

If something in language settings will be changed (like, the plural expression, as in my case, or a project name, or whatever), `get_or_create` function tries to take the object using all kwargs in criteria, finds nothing, tries to create a new language with code `en` and dies with an exception. My modifications make `get_or_create` search by code only